source/macos/AdaptiveCards/AdaptiveCards-bridge/BridgeUtils/BridgeTextUtils.mm

```diff
@@ -8,6 +8,19 @@
 
 @implementation BridgeTextUtils
 
++ (ACSMarkdownParserResult * _Nonnull)processOnRawTextString:(NSString * _Nonnull)textValue hostConfig:(ACSHostConfig * _Nonnull)config
+{
+    auto text = [BridgeConverter getStringCpp:textValue];
+    auto language = [BridgeConverter getStringCpp:@""];
```

Comment:
can you add comment in code why we need to pass empty string in method call.

Reply:
Yes will add it.